### DISCUSSION TITLE
BMS values

### DISCUSSION THREAD

**Discussion by koendruyts** [nathanmarlor/foxess_modbus#170] [General]
I have a Fox ESS H1 
I cannot find any BMS entities after set-up.
In total I have (only) 29 entities and not 39 as I could notice in the integration documentation. Should I add something manual?

**Comment by canton7** [comment]
How are you connected to your inverter? By RS485 or a lan cable plugged directly into the inverter?

Where in the documentation does it say 39? The current version provides 66 over RS485

**Reply by canton7** [reply]
Ah, looking at #162, you're connected directly with a LAN cable.

Unfortunately the H1 doesn't expose the BMS registers over a direct LAN connection (and it doesn't expose charge periods / min/max soc / work mode either). If you need those registers, you'll have to connect over RS485.

I'm curious as to where you found 39 though? I'll fix that.

**Reply by koendruyts** [reply]
Thx for the reply..
39 is found in the image under the text how to configure the integration

"Usage
Modbus Service

Navigate to Settings -> Devices & Services to find...
"

**Reply by canton7** [reply]
Aah fair enough, thanks. That'll be for RS485, and it's quite old (from the first release of the integration, probably).
